### PR TITLE
Course box

### LINKED ISSUE #7: Add Course box
As a developer I would like to have reusable Course box component so that I can define available courses to users.

Design:
https://xd.adobe.com/view/597f1c3c-0b0b-40a9-a2f5-d04530e36939-0c2c/

AC 1:
- [ ] Configurable image with defined aspect ratio, so it takes exact space and does not push content. Image has alt description: "{courseName} course illustration". 

AC 2:
- [ ] Configurable required course name.

AC 3:
- [ ] Configurable required description.

AC 4:
- [ ] Configurable position of image - left or right.

AC 5:
- [ ] Optional link to find more information.

### PR DESCRIPTION
**Pull request by pegak**
Fixes #7.

### CODE REVIEW

File: app/components/CourseBox.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  readonly title: string
+  readonly children: React.ReactNode
+  readonly link?: To
+}
+
+export function CourseBox({ image, side, title, children, link }: Props) {
+  return (
+    <div
+      className={`flex flex-col ${
+        side === "left" ? "md:flex-row" : "md:flex-row-reverse"
```
Comment (xhemalov) [inline app/components/CourseBox.tsx:17]: I do not understand why there is `flex-col` and then `flex-row`.

File: app/components/CourseBox.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      } items-center justify-center gap-x-36 gap-y-4 px-5`}
+    >
+      <img className={`${side === "left" ? "mr-12" : "ml-12"}`} src={image} alt=''></img>
```
Comment (xhemalov) [inline app/components/CourseBox.tsx:20]: Alt is empty.

**Review by xhemalov — CHANGES_REQUESTED** [review]

File: app/components/CourseBox.tsx (hunk 1 above)
Comment (pegak) [reply app/components/CourseBox.tsx:17]: `flex-col` is for mobiles and `flex-row` || `flex-row-reverse` are from medium breakpoint.

File: app/components/CourseBox.tsx (hunk 2 above)
Comment (pegak) [reply app/components/CourseBox.tsx:20]: Great reminder, I'll fix that!

**Review by xhemalov — APPROVED** [review]

**Pull request merged**